lib/project.js

```diff
@@ -11,4 +11,5 @@ class Project {
   }
 }
 
+
```

Comment:
Is this new line needed for linting? If not and there's no other reason, this is probably noise in the PR and can be removed.